src/utils/test/autoref_ptr_test.cpp

```diff
@@ -135,7 +135,7 @@ TEST(RefCountedUnitTest, TestSelfAssignment)
 {
     SelfAssign *p = new SelfAssign;
     dsn::ref_ptr<SelfAssign> var(p);
-    var = var;
+    // var = var;
```

Comment:
I would delete this test forever. But since this test is copied from abseil, we need to keep it the same as it was, in case we need to update from the latest abseil code.